TS_TSX.md

```diff
@@ -0,0 +1,48 @@
+# File Metrics for TypeScript (TS) and TypeScript JSX (TSX)
+
+This document outlines the definition of file metrics for TypeScript (TS) and TypeScript JSX (TSX).
+
+### 1. classes
+
+The "classes" metric counts:
+
+-   keywords like: `class`, `abstract class`, `interface`, `enum`
+-   type aliases with object type literals e.g. `type ObjectType = { message: string};`
```

Comment:
As we currently do not count them when there is more than one object type literal:
```suggestion
-   type aliases with a single object type literal, e.g. `type ObjectType = { message: string};`
```